src/backend/main.ts

```diff
@@ -920,7 +932,10 @@ ipcMain.on('navigate', async (event, appName) => {
 
 ipcMain.handle('getGameInfo', async (event, appName, runner) => {
   // Fastpath since we sometimes have to request info for a GOG game as Legendary because we don't know it's a GOG game yet
-  if (runner === 'legendary' && !LegendaryLibraryManager.hasGame(appName)) {
+  if (
+    !appName ||
```

Comment:
is this to catch empty strings?

Reply:
exactly, we are getting a lot of `app not found in library` because it is coming as empty sometimes, we need to investigate this further and probably avoid the call if no appName on frontend.

Reply:
let's add a TODO, and a logspecifying the early return

```ts
logWarn('skipping get game info because app name is empty')
```